src/PKSim.Presentation/Services/SimulationComparisonTask.cs

```diff
@@ -92,5 +104,27 @@ public void ConfigurePopulationSimulationComparison(PopulationSimulationComparis
             _singleStartPresenterTask.StartForSubject(simulationComparison);
          }
       }
+
+      public bool Delete(IReadOnlyList<ISimulationComparison> simulationComparisons)
```

Comment:
The code was in the command. It's now in a task and the command calls the task. That way we can call the same method from the context menu as well